packages/facade/src/apis/sheets/f-filter.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Nullable, Workbook, Worksheet } from '@univerjs/core';
+import { ICommandService, Inject, Injector } from '@univerjs/core';
+import type { FilterModel, IFilterColumn } from '@univerjs/sheets-filter';
+import type { ISetSheetsFilterCriteriaCommandParams } from '@univerjs/sheets-filter-ui';
+import { ClearSheetsFilterCriteriaCommand, RemoveSheetFilterCommand, SetSheetsFilterCriteriaCommand } from '@univerjs/sheets-filter-ui';
+import type { ISheetCommandSharedParams } from '@univerjs/sheets';
+import { FRange } from './f-range';
+
+/**
+ * This interface class provides methods to modify the filter settings of a worksheet.
+ */
+export class FFilter {
+    constructor(
+        private readonly _workbook: Workbook,
+        private readonly _worksheet: Worksheet,
+        private readonly _filterModel: FilterModel,
+        @Inject(Injector) private readonly _injector: Injector,
+        @ICommandService private readonly _commandSrv: ICommandService
+    ) {
+        // empty
+    }
+
+    /**
+     * Get the filter criteria of a column.
+     * @param col The column number.
```

Comment:
```suggestion
     * @param {number} col The column number.
     
```
if the param is optional, should  like below:
```
 * @param {number} [col] The column number.
```

Reply:
Got it.